package.json

```diff
@@ -1,21 +1,24 @@
 {
   "name": "prebuild-ci",
-  "version": "2.3.0",
+  "version": "3.0.0",
```

Comment:
We'll bump the version on `master`, can you revert this?

Reply:
Of course, done, thank you.